src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/controller/TierCalculationRequiredEventListener.kt

```diff
@@ -23,7 +25,9 @@ class TierCalculationRequiredEventListener(
   @JmsListener(destination = "hmppsoffenderqueue", containerFactory = "hmppsQueueContainerFactoryProxy")
   fun listen(msg: String) {
     val (crn) = getCrn(msg)
-    calculator.calculateTierForCrn(crn)
+    calculator.calculateTierForCrn(crn)?.let {
+      successUpdater.update(crn, it.uuid)
+    }
```

Comment:
This looks fine functionally, but it is a bit confusing that a method called `calculateTierForCrn` only returns a calculation when it has changed, even though it always performs and saves that calculation.